src/mugs.js

```diff
@@ -913,7 +913,8 @@ define([
             calculateAttr: { presence: "notallowed" },
             constraintAttr: { presence: "notallowed" },
             constraintMsgAttr: { presence: "notallowed" },
-            dataValue: { presence: "notallowed" }
+            dataValue: { presence: "notallowed" },
+            requiredAttr: { presence: "notallowed" },
```

Comment:
I'd argue it's a good practice to do trailing commas even though they're unofficial - it's minor, but it makes for slightly less code change and slightly fewer "oops, missed a comma" changes while developing.